eras/babbage/impl/testlib/Test/Cardano/Ledger/Babbage/Imp/UtxowSpec.hs

```diff
@@ -0,0 +1,54 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE OverloadedLists #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Test.Cardano.Ledger.Babbage.Imp.UtxowSpec (spec) where
+
+import Cardano.Ledger.Babbage.Rules (BabbageUtxowPredFailure (..))
+import Cardano.Ledger.Babbage.TxOut (BabbageEraTxOut, referenceScriptTxOutL)
+import Cardano.Ledger.BaseTypes
+import Cardano.Ledger.Coin (Coin (..))
+import Cardano.Ledger.Core
+import Cardano.Ledger.Plutus
+import Lens.Micro
+import Test.Cardano.Ledger.Allegra.ImpTest (produceScript)
+import Test.Cardano.Ledger.Alonzo.Arbitrary (mkPlutusScript')
+import Test.Cardano.Ledger.Alonzo.ImpTest
+import Test.Cardano.Ledger.Imp.Common
+
+spec ::
+  forall era.
+  ( AlonzoEraImp era
+  , BabbageEraTxOut era
+  , InjectRuleFailure "LEDGER" BabbageUtxowPredFailure era
+  ) =>
+  SpecWith (ImpTestState era)
+spec = describe "UTXOW" $ do
+  it "MalformedScriptWitnesses" $ do
```

Comment:
Let's get rid of this one:
* https://github.com/IntersectMBO/cardano-ledger/blob/2c9ba10d085331fa99840a63b40b8e8b716fba5d/libs/cardano-ledger-test/src/Test/Cardano/Ledger/Examples/BabbageFeatures.hs#L1405-L1418
* https://github.com/IntersectMBO/cardano-ledger/blob/2c9ba10d085331fa99840a63b40b8e8b716fba5d/libs/cardano-ledger-test/src/Test/Cardano/Ledger/Examples/BabbageFeatures.hs#L1010-L1043